### ISSUE TITLE
Find Similar Groups and People for a given person or group record

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-marklogic#122]
**Problem Description:**
Must be Implemented with https://github.com/project-lux/lux-frontend/issues/202

We currently do not have a query which can search for "similar groups/people" based off a person or group record. We need to define what "similar groups/people" includes so a query can be created to search the proposed criteria. Some questions to define the query are below(similar to those in #18) 

1. What algorithm do we use to determine the keywords to then incorporate into the similar query, are we only looking within the reference People/Group, and what weight should we give that criteria?
2. What about the People/Group related agents should be considered, and what is the weight of each?
3. What about the People/Group related locations should be considered, and what is the weight of each?
4. What about the People/Group related concepts should be considered, and what is the weight of each?
5. What about the People/Group related objects/works time periods should be considered, and what is the weight of each?
6. Anything else to consider, either directly in the referenced People/Group or related to it? (possibly the object types, related materials, work types) 

**Expected Behavior/Solution:**
TBD 

**Requirements:**
TBD

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- Example endpoints from LUX to use for development and testing, if applicable.

**Dependencies/Blocks:**
- https://github.com/project-lux/lux-frontend/issues/202 (must be implemented concurrently) 

**Related Github Issues:**
- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**
- These links can consist of resources, bugherds, etc.

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by roamye** [comment]
I formed this in a similar manner as  #18 was formed. Please feel free to make any updates. 

cc: @prowns @clarkepeterf

**Comment by roamye** [comment]
From Similar Requirements Meeting ([notes here](https://docs.google.com/document/d/1P0muJ2v1bq-LSNNyLDD2KkZF1LvaSXH7GqSkk4VuBy8/edit?tab=t.0)): 

@clarkepeterf  To-do:
- Add Professional Activity Timespan
- Add Birth/Death Place and Professional Activity Place
- Adjust Weights (testing) 
- Add English Primary Names for Group only

@kkdavis14 to do: 
- provide a list of similar people related to Andy Warhol. Will allow for Peter to work backwards.
- provide other test case examples.

**Comment by roamye** [comment]
12/16 - Core team has decided to pause this work in favor of other priorities(My Collections, Advanced Search, and Unit Portals). All `Similar` tasks have been moved to Future Work.

**Comment by prowns** [comment]
superseded by vector work. closing.
